services/web/server/src/simcore_service_webserver/redis.py

```diff
@@ -64,8 +63,8 @@ async def _create_client(address: str) -> aioredis.Redis:
                     f"{client=}",
                     json.dumps(attempt.retry_state.retry_object.statistics),
                 )
-            assert client  # nosec
-            return client
+        assert client  # nosec
+        return client
```

Comment:
This looks wrong, but this code will get removed by https://github.com/ITISFoundation/osparc-simcore/pull/4055

Reply:
yes already pushed corrected version :+1: